<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udp</class>
 <widget class="QMainWindow" name="udp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>udp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLeft">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>83</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRight">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>83</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnForward">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>83</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>forward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBoxGetHomeLocation">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetHomeLocation">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Get Home Location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHome">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>83</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Control parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBoxMap">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode 0: Manual control</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode 1: Single target</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Target lat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Target lat</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
